server.py

```diff
@@ -834,12 +839,50 @@ def get(self, region):
                 r['id'] = str(r.pop('player'))
                 ranking_list.append(r)
 
+        ranking_criteria = dao.get_region_ranking_criteria(region)
+
         return_dict['ranking'] = ranking_list
+        return_dict['ranking_criteria'] = ranking_criteria
 
         return return_dict
 
+    def put(self, region):
+        dao = Dao(region, mongo_client=mongo_client)
+        args = rankings_criteria_get_parser.parse_args()
+
+        ranking_num_tourneys_attended = int(args['ranking_num_tourneys_attended'])
+        ranking_activity_day_limit = int(args['ranking_activity_day_limit'])
+        #tournament_qualified_day_limit = int(args['tournament_qualified_day_limit'])
+
+        print ranking_num_tourneys_attended
+        print ranking_activity_day_limit
+
+        if not dao:
+            return 'Dao not found', 404
+        user = get_user_from_request(request, dao)
+        if not user:
+            return 'Permission denied', 403
+        if not is_user_admin_for_region(user, region):
+            return 'Permission denied', 403
+
+        try:
+            if ranking_num_tourneys_attended is not None and ranking_activity_day_limit is not None:
+                # TODO Update rankings and store criteria in db
+                dao.update_region_ranking_criteria(region.lower(),
```

Comment:
i did this at the time because I wasn't sure if it would be display_name property. Removing.